AWS - Security Hub - handle a non pagination response of the 'get findings' command (#12898)
demisto/content: Packs/AWS-SecurityHub/ReleaseNotes/1_0_6.md

#### Integrations
##### AWS - Security Hub

> - Fix an issue where non pagination responses of **aws-securityhub-get-findings** command returned no data.

Reviewer: ```suggestion
- Fixed an issue where the ***aws-securityhub-get-findings*** command did not return data in some cases .
```
